airbyte-webapp/src/scss/_z-indices.scss

```diff
@@ -5,3 +5,6 @@ $sidebar: 9999;
 $panelSplitter: 0;
 $dropdownMenu: 2;
 $notification: 20;
+$schemaChangesBackdrop: 3;
```

Comment:
explicitly set a higher z-index than the dropdown menu and the switch (2)

Reply:
Also, unsure if there is a reason not to, but it might be nice to have these ordered by index?  

Happy to make a quick commit that does that...